docroot/themes/humsci/humsci_basic/patterns/hero-text-overlay/hero-text-overlay.html.twig

```diff
@@ -1,11 +1,4 @@
-{%
-  set classes = [
-    'hb-hero-overlay',
-    'hb-hero-overlay--1'
```

Comment:
Ohhhh, was this `hb-hero-overlay--1` class supposed to be a placeholder for "Hero Option 1"? Nice to see us refactoring it out if we don't need it.

Reply:
yep!!